src/workspaces/ShareWorkspaceModal/Collaborator.test.ts

```diff
@@ -206,4 +207,81 @@ describe('a Collaborator component', () => {
     fireEvent.click(writterPermission);
     expect(setAcl).toHaveBeenCalledTimes(1);
   });
+
+  describe('only allows owners and project owners to share with additional permissions', () => {
```

Comment:
Since the 3 test cases here are very similar, this might be a good fit for a parameterized test using `it.each`.
https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
It's now fewer lines with `it.each`, but I'm not sure how much simpler it is. Having the `checkFn` as a parameter feels odd and non-intuitive.

Reply:
Yeah, agreed. Alternatively, the tests could take a boolean parameter and do something like:

```ts
if (shouldDisableCheckbox) {
  expect(checkbox).toHaveAttribute('disabled');
else {
  expect(checkbox).not.toHaveAttribute('disabled');
}
```

Reply:
yeah, I like that better